user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -121,13 +121,12 @@ On Windows, you can use the [Rufus](https://rufus.akeo.ie/) tool to write the
 ISO to a USB key. Be sure to select "Write in DD Image mode" *after* selecting
 the Qubes ISO and pressing "START" on the Rufus main window.
 
-<div class="alert alert-warning" role="alert">
-  <i class="fa fa-exclamation-circle"></i>
-  <b>Note:</b> Choosing this Windows+Rufus method means that you
+<div class="alert alert-info" role="alert">
+  <i class="fa fa-info-circle"></i>
+  <b>Note:</b> Using Rufus to create the installation medium means that you
   <a href="https://github.com/QubesOS/qubes-issues/issues/2051">won't be able</a>
-  to choose the "Test this media & install Qubes" option recommended in the
-  Installation section below. Instead, you'll only be able to choose the
-  "Install Qubes" option.
+  to choose the "Test this media and install Qubes OS" option mentioned in the
+  example below. Instead, choose the "Install Qubes OS" option.
```

Comment:
This doesn't deserve a warning, because the "test this media" option isn't being "recommended" below. As it states at the beginning of that section, it's simply showing you an example with the most common settings.

The previous wording is likely to get users worried over nothing. This alert still might. Many users still won't understand it at all and become terribly confused. I'm of half a mind to remove it, but it's reasonable to let users know that using Rufus will disallow the test option, since they might care about it.

Reply:
@andrewdavidwong I like your wording better, and I agree with you about not getting users worried over nothing. I think your wording definitely lessens potential worry.